db/schema.rb

```diff
@@ -10,11 +10,22 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2022_07_18_160813) do
+ActiveRecord::Schema.define(version: 2022_10_04_145505) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "ad_renewal_letters_exports", id: :serial, force: :cascade do |t|
+    t.date "expires_on"
+    t.string "file_name"
+    t.integer "number_of_letters"
+    t.string "printed_by"
+    t.date "printed_on"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.integer "status", default: 0
+  end
+
```

Comment:
This isn't used any more, it's just the manual letter printing process NCCC previously used, this could be a chore ticket to remove (not for this PR)